### ISSUE TITLE
[BUG] The environment is stored locally and is not sync with the server

### ISSUE THREAD

**Issue by ryanzhang-oss** [oam-dev/kubevela#358]
**Describe the bug**
The environment is stored locally and is not sync with the server. Therefore, it can be deleted locally and not on the serverside or vice versa.

**To Reproduce**
```
~/workspace/oam/kubevela (improvement) $ vela env init vela-test --namespace vela-test --email my@email.com --domain kubevela.io
ENV vela-test CREATED, Namespace: vela-test, Email: my@email.com.
~/workspace/oam/kubevela (improvement) $ kc get ns
NAME              STATUS        AGE
cert-manager      Active        74m
default           Active        75m
demo              Terminating   65m
kube-node-lease   Active        75m
kube-public       Active        75m
kube-system       Active        75m
monitoring        Active        74m
vela-system       Active        74m
vela-test         Active        3s
~/workspace/oam/kubevela (improvement) $ vela env list
NAME     	CURRENT	NAMESPACE
default  	       	default  
demo     	       	demo     
vela-test	*      	vela-test
~/workspace/oam/kubevela (improvement) $ vela env set default
Set env succeed, current env is default, namespace is default
~/workspace/oam/kubevela (improvement) $ vela env delete vela-test
vela-test deleted
~/workspace/oam/kubevela (improvement) $ kc get ns
NAME              STATUS        AGE
cert-manager      Active        74m
default           Active        75m
demo              Terminating   66m
kube-node-lease   Active        75m
kube-public       Active        75m
kube-system       Active        75m
monitoring        Active        74m
vela-system       Active        74m
vela-test         Active        35s
```

**Expected behavior**
The namespace is also deleted.

**Comment by wonderflow** [comment]
It's not a bug, it's by design.

**Comment by zzxwill** [comment]
Related to https://github.com/oam-dev/kubevela/issues/176.

**Comment by wonderflow** [comment]
@Cweiping  can you help fix this issue?

**Comment by Cweiping** [comment]
> @Cweiping can you help fix this issue?

OK.
/assign

**Comment by wonderflow** [comment]
@Cweiping I'm sorry, we should fix #176 instead of this one. Closing it, do not delete namespace when delete a env is by design.
